pyvista/plotting/plotting.py

```diff
@@ -1491,8 +1494,10 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
 
         rgb : bool, optional
             If an 2 dimensional array is passed as the scalars, plot
-            those values as RGB(A) colors. ``rgba`` is also an accepted
-            alias for this.  Opacity (the A) is optional.
+            those values as RGB(A) colors. ``rgba`` is also an
+            accepted alias for this.  Opacity (the A) is optional.  If
+            a scalars array ending with ``"_rgba"`` is passed, this is
+            automatically enabled.
```

Comment:
Should this rather be something like "... the default becomes True"?  The user can still override it.

Reply:
True, that makes more sense.